scripts/deploy.sh

```diff
@@ -174,6 +185,42 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
 
   cd ${OA_DIR}/playbooks/
 
+  #Distribute Magnum configuration files
+  if [[ "${DEPLOY_MAGNUM}" == "yes" ]]; then
+    cat > $OA_OVERRIDES <<'EOF'
+keystone_keystone_conf_overrides:
```

Comment:
Could these variable sections perhaps be moved into config files, rather than inline in the script?